packages/cozy-intent/package.json

```diff
@@ -10,6 +10,7 @@
     "url": "https://github.com/cozy/cozy-libs/issues"
   },
   "dependencies": {
+    "cozy-device-helper": "^2.1.0",
```

Comment:
could you provide a failing job in the PR description?

if we keep this change, could we remove the devDependencies?

Reply:
updated